src/components/Class/ClassCard/index.js

```diff
@@ -378,6 +424,7 @@ function ClassCard({ item, editClass }) {
               <ClassJoinTimerButton
                 startTime={item?.start_time}
                 link={item?.meet_link}
+                setTimeZone={setTimeZone}
```

Comment:
With the suggested renames, we'd have:
```suggestion
                onCanJoin={() => setCanJoin(true)}
```